SCF/lac_scf_diis.dat

```diff
@@ -0,0 +1,143 @@
+import numpy as np
+import math
+
+molecule mol {
+0 1
+O
+H 1 1.1
+H 1 1.1 2 104
+symmetry c1
+}
+
+set{
+    basis aug-cc-pVDZ
+    scf_type pk
+    guess core
+    maxiter 150
+    diis_max_vecs 10
+    diis_min_vecs 2
+    diis_start 1
+    e_convergence 1e-6
+    d_convergence 1e-6
+}
+
+# One-electron integrals and Core-Hamiltonian
+wfn = psi4.new_wavefunction(mol,psi4.get_global_option('BASIS'))
+mints = psi4.MintsHelper(wfn.basisset())
+S = np.asarray(mints.ao_overlap())
+T = np.asarray(mints.ao_kinetic())
+V = np.asarray(mints.ao_potential())
+H = T+V
+
+# Two-electron integrals
+nbf = S.shape[0]
+g_size = (nbf**4)*8/1.e9
+if g_size >1:
+    raise Exception("G tensor is too big!")
+g = np.asarray(mints.ao_eri())
+
+# Number of Doubly Occupied Orbitals
+ndocc = int((sum(mol.Z(A) for A in range(mol.natom()))-mol.molecular_charge())/2)
+
+# Guess Type: GWH or CORE
+if (psi4.get_global_option('GUESS')=='GWH'):
+    F0 = np.zeros((H.shape[0],H.shape[0]))
+    for m in range(H.shape[0]):
+        for n in range(H.shape[0]):
+            F0[m][n] = S[m][n]*0.5*(H[m][m]+H[n][n])
```

Comment:
BTW Psi lies, 0.875 gives you a slightly better result on the average. If you examine the code rather than the comment you will see that Psi actually does this.